src/braket/aws/aws_device.py

```diff
@@ -379,13 +380,21 @@ def _populate_properties(self, session: AwsSession) -> None:
         self._status = metadata.get("deviceStatus")
         self._type = AwsDeviceType(metadata.get("deviceType"))
         self._provider_name = metadata.get("providerName")
-        self._properties = BraketSchemaBase.parse_raw_schema(metadata.get("deviceCapabilities"))
-        device_poll_interval = self._properties.service.getTaskPollIntervalMillis
-        self._poll_interval_seconds = (
-            device_poll_interval / 1000.0
-            if device_poll_interval
-            else AwsQuantumTask.DEFAULT_RESULTS_POLL_INTERVAL
-        )
+        try:
+            self._properties = BraketSchemaBase.parse_raw_schema(metadata.get("deviceCapabilities"))
+            device_poll_interval = self._properties.service.getTaskPollIntervalMillis
```

Comment:
Not in scope but we should look at normalizing time intervals based on resources at some point. It may not make sense for the whole BDK to be normalized to seconds as the default time (pulses, etc.) but maybe within particular resources (tasks or jobs) to have time intervals normalized to avoid conversions.